app/models/event.rb

```diff
@@ -95,4 +87,40 @@ def fetch_github_stats
   def delete_logo
     logo.clear if logo_delete == '1'
   end
+
+  def fetch_attendee_github_stats
+    stats = {}
+
+    event_registrations.find_in_batches(batch_size: 5) do |er_batch|
+      er_batch.each do |er|
+        begin
+          stats.merge!(er.user.email => er.fetch_github_stats)
+        rescue
+          next
+        end
+      end
+
+      sleep(60) # avoid GH search rate limit
+    end
+
+    stats
+  end
+
+  def fetch_project_github_stats
+    stats = {}
+
+    featured_projects.find_in_batches(batch_size: 5) do |fp_batch|
+      fp_batch.each do |fp|
+        begin
+          stats.merge!(fp.project.name => fp.fetch_github_stats)
+        rescue
+          next
+        end
+      end
+
+      sleep(60) # avoid GH search rate limit
```

Comment:
Can you please explicitly specify what the rate limit we're accomodating is here? e.g. `# avoid GitHub search rate limit of 20 requests per minute` (helpful for future refactoring & it's always good to explain any magic numbers in code). Thanks!